profiles/common/modules/features/multisite_forum_community/multisite_forum_community.install

```diff
@@ -30,6 +31,9 @@ function multisite_forum_community_disable() {
   unset($og_context_negotiation_group_context['multisite_forum_community']);
   variable_set('og_context_negotiation_group_context', $og_context_negotiation_group_context);
 
+  /*   * * Remove permissions ** */
```

Comment:
I don't like the way this comment is written, should be more in the lines of
// Remove permissions.

Reply:
Agreed